src/software/world/team_test.cpp

```diff
@@ -551,3 +551,47 @@ TEST_F(TeamTest, update_timestamp_history_from_team_update)
     EXPECT_EQ(three_seconds_future, team.getTimestampHistory()[0]);
     EXPECT_EQ(4, team.getTimestampHistory().size());
 }
+
+TEST_F(TeamTest, get_all_robots_except_goalie)
+{
+    Team team = Team(Duration::fromMilliseconds(2000));
+    Team teamWithoutGoalie = Team(Duration::fromMilliseconds(2000));
+
+    Robot robot_0 = Robot(0, Point(0, 1), Vector(-1, -2), Angle::half(),
+                          AngularVelocity::threeQuarter(), current_time);
+
+    Robot robot_1 = Robot(1, Point(3, -1), Vector(), Angle::zero(),
+                          AngularVelocity::zero(), current_time);
+
+    Robot robot_2 = Robot(2, Point(3, -1), Vector(), Angle::zero(),
+                          AngularVelocity::zero(), current_time);
+
+    team.updateRobots({robot_0, robot_1, robot_2});
+    team.assignGoalie(0);
+    teamWithoutGoalie.updateRobots({robot_1, robot_2});
+
+    EXPECT_EQ(teamWithoutGoalie.getAllRobots(), team.getAllRobotsExceptGoalie());
+}
+
+TEST_F(TeamTest, get_all_robots_except_goalie_1)
+{
+    Team team = Team(Duration::fromMilliseconds(2000));
+    Team teamWithoutGoalie = Team(Duration::fromMilliseconds(2000));
+
+    Robot robot_0 = Robot(0, Point(0, 1), Vector(-1, -2), Angle::half(),
+                          AngularVelocity::threeQuarter(), current_time);
+
+    Robot robot_1 = Robot(1, Point(3, -1), Vector(), Angle::zero(),
+                          AngularVelocity::zero(), current_time);
+
+    Robot robot_2 = Robot(2, Point(3, -1), Vector(), Angle::zero(),
+                          AngularVelocity::zero(), current_time);
+    Robot robot_3 = Robot(3, Point(2, 4), Vector(), Angle::half(),
+                          AngularVelocity::threeQuarter(), current_time);
+
+    team.updateRobots({robot_0, robot_1, robot_2, robot_3});
+    team.assignGoalie(2);
+    teamWithoutGoalie.updateRobots({robot_0, robot_1, robot_3});
+
+    EXPECT_EQ(teamWithoutGoalie.getAllRobots(), team.getAllRobotsExceptGoalie());
```

Comment:
we can probably do without the teamWithoutGoalie, by doing 
```suggestion
    EXPECT_EQ({robot_0, robot_1, robot_3}, team.getAllRobotsExceptGoalie());
```

Reply:
Do I have to go back to my code in my local repo to change it there and commit again? I thought maybe the "commit suggestion" button below the conversation would apply the suggestion for me, but I'm not sure if it did.

Reply:
Yup it did apply it, you can see the changes under Files Changed

<img width="693" alt="image" src="https://user-images.githubusercontent.com/31548846/94478927-d1aeb580-0188-11eb-85c5-ad37b6d5ebb5.png">

Make sure to select "Changes from All Commits"  or select the one commit that you want to check

Generally speaking its better to fix pr comments locally and push because there maybe other instances of similar changes, which the code reviewer might not point out explicitly. But its usually up to the PR authors discretion